graph/db/interfaces.go

```diff
@@ -37,3 +44,323 @@ type NodeTraverser interface {
 	// FetchNodeFeatures returns the features of the given node.
 	FetchNodeFeatures(nodePub route.Vertex) (*lnwire.FeatureVector, error)
 }
+
+// V1Store represents the main interface for the channel graph database for all
+// channels and nodes gossiped via the V1 gossip protocol as defined in BOLT 7.
+type V1Store interface { //nolint:interfacebloat
```

Comment:
Yeah yeah, linter, you're not wrong. But don't judge us on this, we're working on it :joy:

Reply:
😂